packages/typespec-test/test/openai_modular/generated/typespec-ts/src/api/operations.ts

```diff
@@ -18,30 +19,20 @@ import {
 import {
   StreamableMethod,
   operationOptionsToRequestParameters,
-  OperationOptions,
 } from "@azure-rest/core-client";
 import {
   Embeddings,
   Completions,
-  ChatMessage,
   ChatCompletions,
   ImageOperationResponse,
-  ImageSize,
-} from "./models.js";
-
-export interface GetEmbeddingsOptions extends OperationOptions {
-  /**
-   * An identifier for the caller or end user of the operation. This may be used for tracking
-   * or rate-limiting purposes.
-   */
-  user?: string;
-  /**
-   * The model name to provide as part of this embeddings request.
-   * Not applicable to Azure OpenAI, where deployment information should be included in the Azure
-   * resource URI that's connected to.
-   */
-  model?: string;
-}
+} from "../models/models.js";
```

Comment:
I may prefer to import from index.js directly and it is simpler and in case we have some changes happen in index.js.
```suggestion
} from "../models/index.js";
```

Reply:
notification hub is not doing this way https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/notificationhubs/notification-hubs/src/api/cancelScheduledNotification.ts#L6

Reply:
I am ok with this if our index.js would only `export * from xxx.js` no any renaming stuff.